app/dashboard/templates/dashboard/hackathon/new_bounty.html

```diff
@@ -171,7 +171,7 @@ <h1 class="text-center">Fund Prize</h1>
                     </label>
                   {% endif %}
 
-                  <label class="btn btn-radio chain-btn d-flex align-items-center mr-2 mb-2 font-weight-bold py-2 px-4" :class="{'active': chainId === 58}">
+                  <label class="btn btn-radio chain-btn d-flex align-items-center mr-2 mb-2 font-weight-bold py-2 px-4" :class="{'active': chainId === '58'}">
```

Comment:
whats up with this change?

Reply:
@danlipert when polkadot is clicked, it doesn't appear selected (with blue border). this happens because chainId is a string not integer. '58' fixed that.

Reply:
amazing!

Reply:
If you make extra fixes like this, I would say either stick it in a different branch OR be sure to note it in the PR description so everyone knows theres some extra goodness in this PR 👍

Reply:
that's right... a different branch seems cool... since this pr may take a while to be merged